src/Tools/dotnet-monitor/Egress/Extension/ExtensionEgressPayload.cs

```diff
@@ -11,5 +12,6 @@ internal sealed class ExtensionEgressPayload
         public IDictionary<string, string> Properties { get; set; }
         public IDictionary<string, string> Configuration { get; set; }
         public string ProviderName { get; set; }
+        public LogLevel MinimumLogLevel { get; set; }
```

Comment:
This should be attributed such that it is always serialized as a string rather than the underlying value.

Reply:
I'm not sure if I'm interpreting this the right way - do you mean this should be a string, or is there an actual attribute I should be using that dictates its serialization?

Reply:
Use https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonconverterattribute?view=net-8.0 with https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonstringenumconverter?view=net-8.0 on this property.